libsolidity/formal/SMTEncoder.cpp

```diff
@@ -663,6 +663,17 @@ void SMTEncoder::endVisit(FunctionCall const& _funCall)
 	case FunctionType::Kind::Event:
 		// These can be safely ignored.
 		break;
+	case FunctionType::Kind::ObjectCreation:
+	{
+		ArrayType const& arrayType = dynamic_cast<ArrayType const&>(*_funCall.annotation().type);
+		// TODO: support other kinds of arrays
+		if (arrayType.isByteArray())
+		{
+			visitObjectCreation(_funCall);
+			return;
+		}
+		[[fallthrough]];
```

Comment:
Why fallthrough?

Reply:
To have the warning displayed?

Reply:
Ah, for the else case. Was already thinking about the version that supports all arrays :p